### PR TITLE
Analytics docs

### PR DESCRIPTION
**Pull request by robertkeithrippetoe**
Also update for local destroy doc

### CODE REVIEW

**Review by jarededwards — DISMISSED** [review]

**Comment by fharper** [comment]
@robertkeithrippetoe: sorry for the delay, but can you fix the conflicts please? I'll review this PR after :)

**Comment by fharper** [comment]
sorry for the delay @robertkeithrippetoe , I completely forgot about this PR. I added some comments or modifications requests. I also push some commits, please validate that they are fine with you also.

**Review by fharper — CHANGES_REQUESTED** [review]

File: docs/explore/open-telemetry.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### KubeFirst Allows You to Opt out of Data Collection
```
[the remark this answers is not in the document]
Comment (robertkeithrippetoe) [reply docs/explore/open-telemetry.md:1]: Added under mkdocs

File: docs/explore/open-telemetry.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Kubefirst collects data around users in order to optimize future releases. By collecting metrics on what type of clusters are being deployed and how they are being used, Kubefirst prioritizes the features that are being used across the majority of the user base. While we rely on this data to make improvements to the platform, you are always allowed to opt out for any reason.
```
[the remark this answers is not in the document]
Comment (robertkeithrippetoe) [reply docs/explore/open-telemetry.md:3]: Added collected metrics

File: docs/kubefirst/local/destroy.md [hunk 3]
```diff
@@ -22,6 +22,18 @@ To delete your github assets that we created, log into your personal github and
 - gitops repo
 - metaphor-frontend repo
 
+Manual Destroy CLI Example:
```
[the remark this answers is not in the document]
Comment (robertkeithrippetoe) [reply docs/kubefirst/local/destroy.md]: Moved to FAQ

**Review by fharper — APPROVED** [review]
LGTM, good job @robertkeithrippetoe

**Pull request merged**